### PR TITLE
Cscoiva 1124 public views

### PR DESCRIPTION
**Pull request by jpy-valamis**
constants.js localization keys moved to react-intl scheme
Some hardcoded strings localized
Fixed public lupa view section headers to actually display proper headers

### CODE REVIEW

File: src/lomake2.json [hunk 1]
```diff
@@ -0,0 +1,299 @@
+[
```
Comment (rapitkan) [inline src/lomake2.json]: Is this file needed? If so, it should be named better.

File: src/scenes/Jarjestajat/components/LuvatList.js [hunk 2]
```diff
@@ -3,13 +3,14 @@ import * as R from "ramda";
...
 import PropTypes from "prop-types";
+import common from "../../../i18n/definitions/common";
 
 const colWidths = {
   0: "w-4/6",
   1: "w-2/6"
 };
 
-function LuvatList({ history, luvat = [] }) {
+function LuvatList({ history, luvat = [], }) {
```
Comment (rapitkan) [inline src/scenes/Jarjestajat/components/LuvatList.js]: Unneseccary comma found.

File: src/lomake2.json (hunk 1 above)
Comment (jpy-valamis) [reply src/lomake2.json]: I don't know. It's from your commit :)

Comment (rapitkan) [reply src/lomake2.json]: Then you can delete the file.
